backend/connector/models.py

```diff
@@ -62,7 +63,7 @@ class ConnectorMode(models.IntegerChoices):
     connector_mode = models.CharField(
         choices=ConnectorMode.choices,
         default=ConnectorMode.UNKNOWN,
-        db_comment="0: UNKNOWN, 1: FILE_SYSTEM, 2: DATABASE",
+        db_comment="0: UNKNOWN, 1: FILE_SYSTEM, 2: DATABASE, 3: QUEUE",
```

Comment:
@vishnuszipstack do we plan to rename this to `MANUAL_REVIEW` or anything along those lines as @shuveb mentioned?
Also this would need a migration I believe - please check and add the same

Reply:
@chandrasekharan-zipstack done